workflows/eager.nf

```diff
@@ -247,6 +253,14 @@ workflow EAGER {
         ch_versions = ch_versions.mix( PRESEQ_LCEXTRAP.out.versions )
     }
 
+    //
+    // SUBWORKFLOW: aDNA Damage Manipulation
+    //
+
+    MANIPULATE_DAMAGE( ch_dedupped_bams, ch_fasta_for_deduplication.fasta )
```

Comment:
Does this not need a conditon, if any of the three steps are activated, else we don't care?

Reply:
yes